Drupal\views\Plugin\views\display\DisplayPluginBase::validateOptionsForm rejects valid CSS selectors, including '@', which is valid if escaped in CSS.

diff --git a/core/modules/views/src/Plugin/views/display/DisplayPluginBase.php b/core/modules/views/src/Plugin/views/display/DisplayPluginBase.php
index 4fe9d1e204..09dbec1945 100644
--- a/core/modules/views/src/Plugin/views/display/DisplayPluginBase.php
+++ b/core/modules/views/src/Plugin/views/display/DisplayPluginBase.php
@@ -1861,7 +1861,7 @@ public function validateOptionsForm(&$form, FormStateInterface $form_state) {
         break;
       case 'css_class':
         $css_class = $form_state->getValue('css_class');
-        if (preg_match('/[^a-zA-Z0-9-_ ]/', $css_class)) {
+        if (preg_match('/[^a-zA-Z0-9-_@ ]/', $css_class)) {
           $form_state->setError($form['css_class'], $this->t('CSS classes must be alphanumeric or dashes only.'));
         }
         break;
